### DISCUSSION TITLE
magit-delta in forge

### DISCUSSION THREAD

**Discussion by kstatz12** [magit/forge#421] [Support] [answered]
Is there a way to use magit-delta when reviewing pull requests?

**Comment by tarsius** [comment]
I have heard of [magit-delta](https://github.com/dandavison/magit-delta) but haven't checked it out yet, so I don't know. The author of `magit-delta`, @dandavison is in a better position to answer that.

**Reply by kstatz12** [reply, marked as answer]
this is apparently the wrong place for this since its the downstream https://github.com/charignon/github-review that handles it.

**Reply by dandavison** [reply]
Hi @tarsius  and @kstatz12.

@kstatz12 -- I have not yet tried using github-review to review PRs  (and I'm not knowledgable about how it is related to forge / magit). From a technical point of view, magit-delta works by customizing the magit variable `magit-diff-wash-diffs-hook` and therefore as far as I know it should colorize any diff that magit displays, as long as magit calls [`magit-diff-wash-diffs`](https://github.com/magit/magit/blob/3e415b7d533cb0a8faac696a54dbe15ee0d84bea/lisp/magit-diff.el#L2080-L2089) on that diff. See https://github.com/magit/magit/pull/4091#issuecomment-619441921  for the magit change that facilitated this.

If it doesn't seem to work but you feel that it should and you're able to give instructions for reproducing the issue, I'd happy to discuss it in a magit-delta issue.
